sklearn/metrics/_plot/tests/test_precision_recall_display.py

```diff
@@ -48,7 +48,7 @@ def test_precision_recall_display_validation(pyplot):
         # Force `y_true` to be seen as a regression problem
         PrecisionRecallDisplay.from_predictions(y + 0.5, y_pred_classifier, pos_label=1)
     with pytest.raises(ValueError, match=err_msg.format("multiclass")):
-        PrecisionRecallDisplay.from_predictions(y, y_pred_regressor, pos_label=1)
+        PrecisionRecallDisplay.from_predictions(y, y_pred_classifier, pos_label=1)
```

Comment:
Apparently, this change is raising an error. We are expected to pass a 2D-array and we only pass a 1D array. So we should adapt the error message.

Reply:
Yes, I will change the error message.